app/jobs/cleanup.js

```diff
@@ -16,51 +16,54 @@ const CleanupJob = function(config, headerDB, metadataDB, logger) {
 
 /* Delete any file that has not been used in 7 or more days. */
 CleanupJob.prototype.run = function() {
-  let self = this;
-  fs.readdir(self.config.cachePath, function(err, files) {
+  fs.readdir(this.config.cachePath, (err, files) => {
     if (err) {
-      self.logger.error("Could not read the " + self.config.cachePath + " directory.", err);
+      this.logger.error("Could not read the " + this.config.cachePath + " directory.", err);
     } else {
+      this.logger.info("Cleanup job started at " + new Date());
+
       // Iterate over the files in the directory.
-      files.forEach(function(file) {
-        let filePath = path.join(self.config.cachePath, file);
+      files.forEach((file) => {
+        let filePath = path.join(this.config.cachePath, file);
 
         // Delete any unused files with headers and metadata.
         fileSystem.isUnused(filePath, (isUnused) => {
           if (isUnused) {
 
             fileSystem.delete(filePath, (err) => {
               if (err) {
-                self.logger.error(err, filePath);
+                this.logger.error(err, filePath);
               } else {
-                self.logger.info("File deleted", filePath);
+                this.logger.info("File deleted", filePath);
```

Comment:
@rodrigopavezi I added tests for the other logs, which helped me to realize that this wasn't implemented correctly. Please review latest commits. Thx!